docs/mkdocs.yml

```diff
@@ -103,6 +104,7 @@ nav:
       - 'Debugging': config/troubleshooting/debugging.md
     - 'Mail Delivery with POP3': config/pop3.md
     - 'Advanced Configuration':
+        - 'Additional, user-supplied custom configurations': config/advanced/user-patches.md
```

Comment:
I don't think we want `,` in nav list entries? This is also quite a long entry name that it probably won't fit a single line on the nav sidebar.

If you need to provide more context, create a subdirectory and move the document page into that. Then like the parent `Advanced Configuration`, you can provide a group/category name (has no relation to directory layout), followed by any children pages (single child is fine, just like `Maintenance`).

It might be better placed in `Advanced Configuration --> Override the Default Configs`, under the name "Modifications via Script` or similar. That category already details similar functionality specific to Dovecot or Postfix.

Reply:
You're right. I've placed it under `Advanced Configuration --> Override the Default Configs` and renamed it to "Modifications via Script".